src/components/base/BaseButton.vue

```diff
@@ -137,11 +137,9 @@ export default defineComponent({
     border-color: $red-cherryred;
     font-weight: $font-weight-regular;
 
-    @media (hover: hover) {
-      &:hover {
-        background-color: $red-totempole;
-        border-color: $red-totempole;
-      }
+    @include hover {
```

Comment:
Using mixin for hover seems a bit overcomplicated to me 🤔

why not just `&:hover { ... }` here and in all other places?

Reply:
I also included `@media (hover: hover) { ... }` query in that mixin, thinking that would be useful to check if user's device's interface includes hover state. This snippet got a little bit big for me for such a small thing - 3 times "hover" keyword just to create hover effect:
```
@media (hover: hover) {
    &:hover {
      ...
    }
 }
  ```
  And I use it so often that copying the snippet did not feel so nice.
  
  What do you think? Should I switch it back regardless of the above?

Reply:
But what problem you are trying to solve by adding `@media (hover: hover) { ... }`?

I'm honestly curious! I know what it does but tbh afair personally I've never used it 😅

Reply:
Thanks for explanation - I know this behavior actually but never considered it a problem as in most cases buttons/links and other "hoverable" elements are actually here to start some interactions so it's often not possible to see this behavior as a user. In some cases I even consider it a feature 😄 ofc it's not a blocker! :shipit: